mod_modular_vanilla/hooks/skills/skill.nut

```diff
@@ -201,7 +201,9 @@
 			toHit = ::Math.floor(toHit * (1.0 - this.MV_getDiversionChance(_targetEntity, _propertiesForUse, _propertiesForDefense)));
 		}
 
-		return this.Math.max(::Const.Combat.MV_HitChanceMin, this.Math.min(::Const.Combat.MV_HitChanceMax, toHit));
+		local hitChanceMin = ::Math.max(0, ::Math.min(_propertiesForUse.MV_HitChanceMin, _propertiesForDefense.MV_HitChanceMin));
+		local hitChanceMax = ::Math.min(100, ::Math.max(_propertiesForUse.MV_HitChanceMax, _propertiesForDefense.MV_HitChanceMax));
+		return ::Math.max(hitChanceMin, ::Math.min(hitChanceMax, toHit));
```

Comment:
Question: Should we use `::Math.min` or `::Math.max` here for each of the two values from the `propertiesForUse` and `propertiesForDefense`?
- If we use `::Math.min` for `MV_HitChanceMin` then it means that if one skill/attacker sets the `MV_HitChanceMin` to 90 and the another skill/defender sets the `MV_HitChanceMin` to 100 then it will be set to 90. If we use `::Math.max` then it will be set to 100.
- Similarly for the other one.
Which logic is more favorable?

Reply:
Do I understand correctly that with default 95, if you use min then once can't achieve 100 without hooking both sides? And if one side is hooked by somebody then hooking another side is broken for everyone else? It looks broken either way.

I would use set/unset instead of default + min/max, i.e. by default those `MV_HitChanceMin` and `...Max` is set to null everywhere. Then 5/95 global defaults, which you can make some MV_* global vars. If any of them set then it's used instead of defaults. If both props set some value to non-null then something takes precedence, say attacker's propertiesForUse.